docs/getting-started.md

```diff
@@ -66,7 +68,7 @@ Make sure your build host meets the following requirements:
 - 50 Gbytes of free disk space
 - Runs a supported Linux distribution
 
-Currently, the Yocto Project is supported on a number of Linux Distributions. This guide covers only for Ubuntu 18.04 (LTS). To know more about the complete list of    supported Linux distributions, please visit [Supported Linux Distributions](https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#detailed-supported-distros)
+Currently, the Yocto Project is supported on a number of Linux Distributions. This guide covers only for Ubuntu 22.04 (LTS). To know more about the complete list of    supported Linux distributions, please visit [Supported Linux Distributions](https://www.yoctoproject.org/docs/current/mega-manual/mega-manual.html#detailed-supported-distros)
 - Git 1.8.3.1 or greater
```

Comment:
Update git, tar, gcc and others version based on latest requirement
https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#compatible-linux-distribution